docs/howto/environment.md

```diff
@@ -2,11 +2,11 @@
 
 1. Create a clean conda install:
   ```bash
-  conda env create -n my-example-env python=3.7
+  conda create -n my-example-env python=3.7
   conda activate my-example-env
   ```
 
-2. Ensure that you install node-js using conda first*. If you allow pip to
+2. Ensure that you install `node-js` using conda first*. If you allow pip to
```

Comment:
`nodejs` or node-js. 
`conda install node-js` will fail.

Reply:
I saw this node-js, and I considered it to be the project name (which can be node, node-js, node.js, NodeJS, etc). For the conda package I think it's `nodejs` - even though there is a `node` package in conda too. Just to confuse people

Reply:
Changed to `nodejs` for clarity of context with Conda here.